xbmc/guilib/GUIVisualisationControl.h

```diff
@@ -95,5 +95,5 @@ class CGUIVisualisationControl : public CGUIControl, public IAudioCallback
   std::string m_presetsPath; /*!< To add-on sended preset path */
   std::string m_profilePath; /*!< To add-on sended profile path */
 
-  ADDON::CVisualization* m_instance{nullptr};
+  KODI::ADDONS::CVisualization* m_instance{nullptr};
```

Comment:
`#include "addons/Visualization.h"` can be moved to the cpp file and this class forward-declared.

Reply:
Thanks for the note, will change it.
Also you brought me to an idea to change it to `std::unique_ptr` to confirm destruct in all cases.